packages/klevu-core/src/connection/responseQueryObject.ts

```diff
@@ -308,6 +313,43 @@ export class KlevuResponseQueryObject {
       }
 
       case "kmcRecommendation": {
+        if (!this.func.params?.kmcConfig) {
+          break
+        }
+
+        const config = this.func.params?.kmcConfig
+
+        if (config.metadata.action === "HIDE_RECOMMENDATION") {
+          break
+        }
+
+        if (config.metadata.action === "STATIC_CONTENT") {
+          if (!config.staticContent) {
+            break
+          }
+          this.recommendationBannerClickEvent = function ({ resolution }) {
+            const image = config.staticContent?.[0].image.find(
+              (image) => image.resolution === resolution
```

Comment:
just pointing out, the mobile resolution if not present is replaced with desktop resolution image, should we check for that here?